airflow-core/src/airflow/api_fastapi/execution_api/routes/task_instances.py

```diff
@@ -564,8 +568,86 @@ def get_previous_successful_dagrun(
     return PrevSuccessfulDagRunResponse.model_validate(dag_run)
 
 
-@router.only_exists_in_older_versions
-@router.post(
+@router.get("/count", status_code=status.HTTP_200_OK)
```

Comment:
We will need to add cadwyn migration for the new endpoints: https://docs.cadwyn.dev/concepts/version_changes/

Reply:
I don't think so. We only need to add a migration for breaking changes (or changes to existing endpoints) from what I understand.

Reply:
Same as https://github.com/apache/airflow/pull/48651#discussion_r2029775166

Reply:
Check https://github.com/apache/airflow/pull/48651#discussion_r2029791979 :)